### ISSUE TITLE
How to comment, engage with your systemd-boot Post?

### ISSUE THREAD

**Issue by superuser-miguel** [hboetes/wiki#1]
There are several things I'd like to note in you wiki post about systemd-boot, is the issue tracker enough?

**Comment by hboetes** [comment]
Sure. A diff would be ideal.

**Comment by superuser-miguel** [comment]
Well, first off, this approach will block the user from updating the system through `dnf system-upgrade` so for every upgrade they will have to re do this if they want to ugrade to a new release.

Having the `/efi` as the ESP I believe is the culprit. but having the ESP as `/boot/efi` will work. You just need to modify some steps to achieve this.

**Comment by hboetes** [comment]
My `dnf system-upgrade`s went fine IIRC. Why would it not work?

**Comment by superuser-miguel** [comment]
>Why would it not work?

I've done this through the years now, and in the past it did not work. If it is working now then cool. I made modifications over the years to assure the user would not be blocked from `system-upgrades` so there was no need for me to look back.

**Comment by hboetes** [comment]
Alright, let's keep this issue open, to keep it in mind. I'll upgrade to 41 one of these days anyway.

**Comment by superuser-miguel** [comment]
>I'll upgrade to 41 one of these days anyway.

You can always try it in a VM as well. :+1:

**Comment by hboetes** [comment]
Yeah, good idea. And now that I think about it, the upgrade did get stuck, and I did have to reinstall. Silly, I didn't take notes.

**Comment by hboetes** [comment]
Alright, the upgrade did work, but it installed the kernel in /boot/efi etc. Let's try again whilst using /boot/efi rather than /efi

**Comment by hboetes** [comment]
It's back to /boot/efi

And I added a few more tidbits. Anything else you spotted?

**Comment by dklann** [comment]
@hboetes, I found your github wiki and notes from [this Fedora Discussion post](https://discussion.fedoraproject.org/t/how-to-switch-to-systemd-boot-after-dnf-system-upgrade/98739/20). I followed your instructions (on Fedora 41) and it worked first time through without issues (I'm using /boot/efi, not /efi because it was already a mounted file system)! Thanks for keeping and publishing your detailed notes about switching to systemd-boot!
